lesson04/assignment/css/style.css

```diff
@@ -0,0 +1,362 @@
+/* Colors
+#ff8c00 // darkorange
+#fffff0 // ivory
+#dcdcdc // gainsboro light gray
+#696969 // dimgray
+#191970 // midnightblue
+#f0f8ff // aliceblue
+#000000 // black
+*/
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+
+    width: 850px;
+    font-family: Arial, Helvetica, sans-serif;
+    /* font-size: 1.2em; */
+    line-height: 1.4;
+}
+/*
+.overlapping-header {
+    padding: 0.5em;
+    margin-top: 1em;
+    margin-left: 1.5em;
+    width: 8em;
+    position: relative;
+    z-index: 1;
+    color: yellow;
+    background-color: #333;
+    text-align: center;
+    text-transform: uppercase;
+}
+
+.quotations {
+    margin-top: -1em;
+    padding: 1.5em 1.5em 0.5em 1.5em;
+    color: #eee;
+    background-color: #999;
+}
+
+blockquote {
+    margin-bottom: 1em;
+}
+*/
+
+.containerA {
+    height: 100vh;
+    width: 100vw;
+    display: grid;
+    grid-template-columns: repeat(5, 1fr);
+    grid-template-rows: 450px 1fr 450px 1fr 200px;
+    grid-template-areas:
```

Comment:
I like your use of css grid.  Is is great for controlling the overall page layout.